internal/commands/user/list/flags.go

```diff
@@ -0,0 +1,65 @@
+package list
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	flagState      = "state"
+	flagStateShort = "s"
+	flagStateUsage = `select the state of users to list, available options: ["enabled", "disabled"]`
+
+	flagStatus      = "status-pending"
+	flagStatusShort = "p"
+	flagStatusUsage = `select the state of users to list, available options: ["enabled", "disabled"]`
+
+	flagProviderTypes      = "provider"
+	flagProviderTypesShort = "t"
+	flagProviderTypesUsage = `todo add description`
+
+	flagUsers      = "users"
+	flagUsersShort = "u" //idk what you want this to be
+	flagUsersUsage = `todo add description`
+)
+
+const (
+	providerTypeLocalUserPass string = "local-userpass"
+	providerTypeAPIKey        string = "api-key"
+	providerTypeFacebook      string = "oauth2-facebook"
+	providerTypeGoogle        string = "oauth2-google"
+	providerTypeAnonymous     string = "anon-user"
+	providerTypeCustom        string = "custom-token"
+)
+
+var (
+	errInvalidProviderType = func() error {
+		allProviderTypes := []string{
```

Comment:
if you leave these typed as strings, then you won't need to do the `func() error` approach here and can simply define the error inline:

```go
errInvalidProviderType = fmt.Errorf(
  "unsupported value, use one of [%s] instead",
  strings.Join(
    []string{providerTypeAPIKey, providerTypeFacebook, ...},
    ", ",
  ),
)
```